packages/account/imAccount/index.test.ts

```diff
@@ -0,0 +1,54 @@
+import config from "~config/test"
+import {
+  createValidator,
+  ValidatorType
+} from "~packages/account/imAccount/validator"
+import number from "~packages/util/number"
+import { describeAccountSuite } from "~packages/util/testing/suite/account"
+
+import { imAccount } from "./index"
+
+describeAccountSuite(
+  "imAccount",
+  () => {
+    const ecdsaValidator = createValidator(
```

Comment:
I think we can new the ECDSA validator directly here. Since each validator would have quite different arguments to construct, a common util function like `createValidator` is hard to have a suitable interface for each validator.